lib/ansible/modules/system/pid.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: pid
+version_added: 2.8
+description: "Gives the pid of the given process name"
+short_description: "Gives the pid of the given process name"
+author:
+  - Saranya Sridharan
+requirements:
+  - only standard library needed
```

Comment:
That's not true. You need an external program called `pidof`. This is *not* installed on every Unix (see http://macappstore.org/pidof/).